tests/storage.test.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var request = require('supertest');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['request' is defined but never used.](https://www.codacy.com/app/jasoncalabrese/cgm-remote-monitor/file/759804173/issues/source?bid=2449627&fileBranchId=2449709#l3)